### DISCUSSION TITLE
Date stamp keep when copy to/from ftp

### DISCUSSION THREAD

**Discussion by PeZe42** [doublecmd/doublecmd#1271] [Suggestions]
Hi from germany.
it would be good if this were preserved.

Double Commander
Version: 0.8.3 beta
Revision: 8160
Build date: 2018/06/05
Lazarus: 1.8.4.0
FPC: 3.0.4
Platform: x86_64-Linux-gtk2
OS version: Debian 12.0

Widgetset library: GTK 2.24.33

**Comment by alexx2000** [comment]
Hi, version 0.8.3 is obsolete and unsupported. DC preserves time when FTP-server has `MFMT` command support.

**Comment by PeZe42** [comment]
Ok, thanks, I will tried it.
MFMT, ok found
"Server" is fritz.box from AVM

**Comment by PeZe42** [comment]
Won't work with the german Fritzbox from AVM.

Thanks!

**Comment by ciros88** [comment]
it is possible to add a "preserve timestamps" (like filezilla)  checkbox on ftp connection advanced settings? thanks

**Reply by alexx2000** [reply]
No, it is determined by copy dialog options.

**Reply by ciros88** [reply]
yeah, the proposal is to add a preserve timestamp setting as property of the single ftp connection as override when copying from it. the scenario is that users want to preserve only timestamp of files for specific ftp source , without needing to switch the global timestamp preservation every time. seems reasonable to me, isn't it?
